lib/unifex/native_code_generator.ex

```diff
@@ -47,6 +49,35 @@ defmodule Unifex.NativeCodeGenerator do
   * `I` indents every line of string
   """
   @spec sigil_g(String.t(), charlist()) :: String.t()
+  def sigil_g(content, 'j(' ++ flags) when is_list(content) do
+    {joiner, ')' ++ flags} = flags |> Enum.split_while(&([&1] != ')'))
+    content = content |> Enum.join("#{joiner}")
+    sigil_g(content, flags)
+  end
+
+  def sigil_g(content, 'n' ++ flags) when is_list(content) do
```

Comment:
BTW, why on earth would you apply sigils on list? Not only it contradicts the specs, but also is super weird. Why don't you extract the functionality for lists into separate function?